EDU-17893: Update CMS integration instructions (#2570)
vtexdocs/dev-portal-content: docs/faststore/docs/getting-started/2-starting-the-project.mdx

| ----------- | ------------------------------------------------------ |
| **Catalog** | Choose which catalog your store should be linked with. |

## Step 2 - Setting up integrations

On this page, you'll set up the native CMS integration and connect to a GitHub organization.

### CMS Integration

Configure the native [CMS](https://developers.vtex.com/docs/guides/cms-for-faststore-storefronts) integration for your store.

**Select a store**

Choose whether to create a new CMS store or connect to an existing one:

- **New store** (default): Creates a new CMS store for your FastStore project. Fill in the following fields:

>   ![new-store-cms](https://vtexhelp.vtexassets.com/assets/docs/src/cms-integrations-new___52da60694ecf236cffa8630c2ab45c17.png)

Author: ```suggestion
<!-- ![new-store-cms](https://vtexhelp.vtexassets.com/assets/docs/src/cms-integrations-new___52da60694ecf236cffa8630c2ab45c17.png) -->
```

  | Field | Description |
  | ----- | ----------- |
  | **Store ID** | Unique identifier for the store (e.g., `my-store`). |

>   | **Store name** | Display name for the store (e.g., `My Store`). |

Author: ```suggestion
  | **Store ID** | Unique identifier for the store (for example, `my-store`). |
  | **Store name** | Display name for the store (for example, `My Store`). |
```

>   | **Store name** | Display name for the store (e.g., `My Store`). |
>   | **Build type** | Choose between the following options: <ul><li><b>FastStore WebOps</b> (default): Specifies that store builds will be performed by FastStore WebOps.</li><li><b>No build:</b> Specifies that the store will not trigger builds on any system.</li><li><b>External build system:</b> Specifies that the store build will be performed by an external system. When choosing this type, a new field becomes visible to specify the Webhook URL (required for this build type), which will be the endpoint to be called to perform the store build.</li></ul> |
> 

Author: ```suggestion
  | **Build type** | Choose between the following options: <ul><li><b>FastStore WebOps</b> (default): Specifies that store builds will be performed by FastStore WebOps.</li><li><b>No build:</b> Specifies that the store will not trigger builds on any system.</li><li><b>External build system:</b> Specifies that the store build will be performed by an external system. When choosing this type, a new field becomes visible to specify the Webhook URL (required for this build type), which will be the endpoint to be called to perform the store build.</li></ul> |
 | **Default locale configuration** | The Default locale configuration section lets you define the primary locale for your store by setting a Name (label for this locale, for example, Portuguese (BR)), Language (content language, for example, Portuguese (pt)), and Region (country/market, for example, Brazil (BR)). This locale is used as the default context for your CMS content and storefront when the store is created. For more information about the available languages and regions, see [Supported languages and regions](https://help.vtex.com/en/docs/tutorials/supported-languages-and-regions). |

```

| ----------- | ------------------------------------------------------ |
| **Catalog** | Choose which catalog your store should be linked with. |

## Step 2 - Setting up integrations

On this page, you'll set up the native CMS integration and connect to a GitHub organization.

### CMS Integration

Configure the native [CMS](https://developers.vtex.com/docs/guides/cms-for-faststore-storefronts) integration for your store.

**Select a store**

Choose whether to create a new CMS store or connect to an existing one:

- **New store** (default): Creates a new CMS store for your FastStore project. Fill in the following fields:

> <!-- ![new-store-cms](https://vtexhelp.vtexassets.com/assets/docs/src/cms-integrations-new___52da60694ecf236cffa8630c2ab45c17.png) -->

Author: ```suggestion
{/* ![new-store-cms](https://vtexhelp.vtexassets.com/assets/docs/src/cms-integrations-new___52da60694ecf236cffa8630c2ab45c17.png) */}
```

  | Field | Description |
  | ----- | ----------- |
  | **Store ID** | Unique identifier for the store (for example, `my-store`). |
  | **Store name** | Display name for the store (for example, `My Store`). |
  | **Build type** | Choose between the following options: <ul><li><b>FastStore WebOps</b> (default): Specifies that store builds will be performed by FastStore WebOps.</li><li><b>No build:</b> Specifies that the store will not trigger builds on any system.</li><li><b>External build system:</b> Specifies that the store build will be performed by an external system. When choosing this type, a new field becomes visible to specify the Webhook URL (required for this build type), which will be the endpoint to be called to perform the store build.</li></ul> |

>  | **Default locale configuration** | The Default locale configuration section lets you define the primary locale for your store by setting a Name (label for this locale, for example, Portuguese (BR)), Language (content language, for example, Portuguese (pt)), and Region (country/market, for example, Brazil (BR)). This locale is used as the default context for your CMS content and storefront when the store is created. For more information about the available languages and regions, see [Supported languages and regions](https://help.vtex.com/en/docs/tutorials/supported-languages-and-regions). |

Author: ```suggestion
 | **Default locale configuration** | The Default locale configuration section lets you define the primary locale for your store by setting a Language (content language, for example, Portuguese (pt)), and Region (country/market, for example, Brazil (BR)). This locale is used as the default context for your CMS content and storefront when the store is created. For more information about the available languages and regions, see [Supported languages and regions](https://help.vtex.com/en/docs/tutorials/supported-languages-and-regions). |
```
